unittests/ThunkLibs/generator.cpp

```diff
@@ -415,9 +415,15 @@ TEST_CASE_METHOD(Fixture, "UnknownAnnotation") {
 TEST_CASE_METHOD(Fixture, "VersionedLibrary") {
   const auto output = run_thunkgen_host("", "template<auto> struct fex_gen_config { int version = 123; };\n");
 
+#if CLANG_VERSION_MAJOR >= 19
```

Comment:
The renamed API was added in LLVM 17 (in addition to deprecating the one they removed in LLVM 19).

```suggestion
#if CLANG_VERSION_MAJOR >= 17
```